src/features/advanced-content-insights/ContentInsightsSection.js

```diff
@@ -0,0 +1,145 @@
+// @flow
+
+import React from 'react';
+import type { Node } from 'react';
+import noop from 'lodash/noop';
+import { FormattedMessage } from 'react-intl';
+import messages from '../unified-share-modal/messages';
+import { bdlGray50 } from '../../styles/variables';
+import Checkbox from '../../components/checkbox';
+import DropdownMenu from '../../components/dropdown-menu';
+import Gear from '../../icon/line/Gear16';
+import IconInfo from '../../icons/general/IconInfo';
+import { Menu, MenuItem } from '../../components/menu';
+import PlainButton from '../../components/plain-button';
+import Toggle from '../../components/toggle';
+import Tooltip from '../../components/tooltip';
+import { ITEM_TYPE_FILE } from '../../common/constants';
+
+import type { item as itemtype } from '../unified-share-modal/flowTypes';
+import type { contentInsightsConfigType } from './flowTypes';
+
+type Props = {
+    contentInsightsConfig?: contentInsightsConfigType,
+    item: itemtype | null,
+    onAdvancedInsightsEmailToggle?: (isEnabled: boolean) => void,
+    onAdvancedInsightsNotificationToggle?: (isEnabled: boolean) => void,
+    onAdvancedInsightsToggle?: (isEnabled: boolean) => void,
+    renderOptionsInDropdown?: boolean,
+    submitting: boolean,
+};
+
+type CheckboxWrapperProps = {
+    children: Node,
```

Comment:
if you do `React.Node` does it prevent having to do the extra import of `import type { Node } from 'react';`?